pkg/metrics/statsd/e2e/metrics.go

```diff
@@ -0,0 +1,53 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+)
+
+const (
+	DimensionARMGeoLocation = "armGeoLocation"
+	DimensionARMResourceID  = "armResourceID"
+	DimensionResourceType   = "resourceType"
+	DimensionSucceeded      = "Succeeded"
+
+	LogEntryIsE2EEmittableMetric = "IsE2EEmittableMetric"
+	LogEntryMetricName           = "MetricName"
+	LogEntryMetricStatus         = "MetricStatus"
+)
+
+type E2ELogToMetrics struct {
+	m metrics.Emitter
```

Comment:
Give this a descriptive name instead of a single letter.
```suggestion
	metricsEmitter metrics.Emitter
```

Reply:
Updated name.